### DISCUSSION TITLE
Custom cursor broken when using smoothscrollbar.js

### DISCUSSION THREAD

**Discussion by gjwjr** [idiotWu/smooth-scrollbar#486] [Q&A] [answered]
I am using a custom cursor that changes when hovering over images, this works fine in a normal situation, but when also using 'smoothscrollbar.js' the cursor does not move down with the page when you scroll. You can see how it is supposed to work [here](https://codepen.io/gabrielcojea/pen/jOWRBrL), but [this](https://codepen.io/gwjr/pen/NWYaBJV/40c5154700d8db2a14990b139e5f6295) is what is happening when using smoothscrollbar.js.

Does anyone know how to fix this issue?

**Comment by sadeghbarati** [comment, marked as answer]
Init `smooth-scrollbar` on another element instead of **body**

```html
<body>
    <main data-scrollbar>
    ...
    </main>

    <!-- cursor outside of smooth-scrollbar element -->
    <div id="custom-cursor"></div>
</body>
```


Check [Cuberto](https://cuberto.com/) which is using `smooth-scrollbar` and custom cursor
